Package/ExcelDna.AddIn/content/ExcelDna-Template.dna

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <DnaLibrary Name="%ProjectName% Add-In" RuntimeVersion="v4.0" xmlns="http://schemas.excel-dna.net/addin/2018/05/dnalibrary">
-  <ExternalLibrary Path="%OutputFileName%" ExplicitExports="false" LoadFromBytes="true" Pack="true" />
+  <ExternalLibrary Path="%OutputFileName%" ExplicitExports="false" LoadFromBytes="true" Pack="true" IncludePdb="true" />
```

Comment:
I'm not sure this should be the default setting for a new project based on the NuGet package.

Reply:
I dont't have a strong feeling here. It seems the only cost is a few more Kb, which is reasonable, and the upside is to show the feature (but this could also be achieved with the attribute present with its default value false).